app/code/Magento/Cookie/Test/Mftf/Test/StorefrontVerifySecureCookieTest.xml

```diff
@@ -22,26 +22,24 @@
             <group value="secure_storefront_url"/>
         </annotations>
         <before>
-            <amOnPage url="/" stepKey="goToHomePage"/>
+            <!-- Go to storefront home page -->
+            <actionGroup ref="StorefrontOpenHomePageActionGroup" stepKey="openStoreFrontHomePage"/>
             <executeJS function="return window.location.host" stepKey="hostname"/>
+            <!-- Login as admin -->
+            <actionGroup ref="AdminLoginActionGroup" stepKey="loginAsAdmin"/>
             <magentoCLI command="config:set web/unsecure/base_url https://{$hostname}/" stepKey="setUnsecureBaseURL"/>
             <magentoCLI command="config:set web/secure/base_url https://{$hostname}/" stepKey="setSecureBaseURL"/>
             <magentoCLI command="config:set web/secure/use_in_frontend 1" stepKey="useSecureURLsOnStorefront"/>
             <actionGroup ref="CliCacheFlushActionGroup" stepKey="flushCache">
-                <argument name="tags" value=""/>
+                <argument name="tags" value="config full_page"/>
             </actionGroup>
         </before>
         <after>
-            <amOnPage url="/" stepKey="goToHomePage"/>
-            <executeJS function="return window.location.host" stepKey="hostname"/>
-            <magentoCLI command="config:set web/unsecure/base_url http://{$hostname}/" stepKey="setUnsecureBaseURL"/>
```

Comment:
could you please explain why you removed these steps? aren't they suppose to revert the settings set in the "before" tag?